ansys/dpf/core/meshes_container.py

```diff
@@ -37,6 +39,63 @@ def __init__(self, meshes_container=None, server=None):
             self, types.meshed_region, collection=meshes_container, server=self._server
         )
 
+    def plot(self, fields_container=None, **kwargs):
+        """Plot the meshes container with a specific result if
+        fields_container is specified.
+
+        Parameters
+        ----------
+        fields_container : FieldsContainer, optional
+            Data to plot. The default is None.
+
+        Examples
+        --------
+        >>> from ansys.dpf import core as dpf
+        >>> from ansys.dpf.core import examples
+        >>> model = dpf.Model(examples.multishells_rst)
+        >>> mesh = model.metadata.meshed_region
+        >>> split_mesh_op = dpf.Operator("split_mesh")
+        >>> split_mesh_op.connect(7, mesh)
+        >>> split_mesh_op.connect(13, "mat")
+        >>> meshes_cont = split_mesh_op.outputs.mesh_controller()
+        >>> disp_op = dpf.Operator("U")
+        >>> disp_op.connect(7, meshes_cont)
+        >>> ds = dpf.DataSources(examples.multishells_rst)
+        >>> disp_op.connect(4, ds)
+        >>> disp_fc = disp_op.outputs.fields_container()
+        >>> meshes_cont.plot(disp_fc)
+
+        """
+        notebook = kwargs.pop("notebook", None)
```

Comment:
Give the user the option to disable showing edges, but make it default ``True``.
```suggestion
        kwargs.setdefault("show_edges", True)
        notebook = kwargs.pop("notebook", None)
```